www/js/localstorage.js

```diff
@@ -0,0 +1,36 @@
+'user strick';
+
+angular.module('emission.main.localstorage',['angularLocalStorage'])
```

Comment:
Should I make this module specific to calorie data or this broad local storage module better so we can add factories to this module later.